{
  "Faults": [
    {
      "latitude": 51.25871746188038,
      "type": "LV",
      "reference": "MR9302",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-10-20T20:35:00.000+0000",
      "affectedCustomerCount": 22,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "44ade973-ae5d-403f-bf01-39a8c2b19676",
      "minLongitude": -1.0963,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.093015128073407,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-10-20T17:00:00.000+0000",
      "estimatedRestoration": "2023-10-22T02:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MR9302'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MR9302",
      "maxLongitude": -1.0911,
      "minLatitude": 51.2577,
      "name": "RG21 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG21 3ND",
        "RG21 3NE"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.0963,
              51.2594
            ],
            [
              -1.0911,
              51.2594
            ],
            [
              -1.0911,
              51.2577
            ],
            [
              -1.0963,
              51.2577
            ],
            [
              -1.0963,
              51.2594
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-10-20T21:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.2594,
      "etrUpdate": true
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-10-21T14:22:00.000+0000",
      "latitude": 51.56220913839558,
      "estimatedRestoration": "2023-10-22T11:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MS0412'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MS0412",
      "reference": "MS0412",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-10-21T15:05:00.000+0000",
      "affectedCustomerCount": 18,
      "name": "SN3 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SN3 5DN",
        "SN3 5DP",
        "SN3 5EA",
        "SN3 5EB",
        "SN3 5EU"
      ],
      "location": {
        "coordinates": [
          -1.7209710992014937,
          51.56220913839558
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2023-10-21T15:10:00.000+0000",
      "UUID": "7abe86e4-6ae3-4fd7-b49e-4522fc79ef26",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "etrUpdate": true,
      "longitude": -1.7209710992014937,
      "resolved": false
    },
    {
      "latitude": 51.29206345372262,
      "type": "LV",
      "reference": "MS0384",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-10-21T16:15:00.000+0000",
      "affectedCustomerCount": 31,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "83049a4f-c81c-4e12-8bbf-7deee77ec878",
      "minLongitude": -1.0684,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.0514567922402758,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-10-21T14:00:00.000+0000",
      "estimatedRestoration": "2023-10-22T02:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MS0384'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MS0384",
      "maxLongitude": -1.0413,
      "minLatitude": 51.2858,
      "name": "RG24 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG24 8JY",
        "RG24 8LN",
        "RG24 8LP",
        "RG24 8LT",
        "RG24 8RD",
        "RG24 8RE",
        "RG24 8RG",
        "RG24 8YA",
        "RG27 0SQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.0684,
              51.3023
            ],
            [
              -1.0413,
              51.3023
            ],
            [
              -1.0413,
              51.2858
            ],
            [
              -1.0684,
              51.2858
            ],
            [
              -1.0684,
              51.3023
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-10-21T15:15:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.3023,
      "etrUpdate": true
    },
    {
      "latitude": 51.74570891442476,
      "type": "HV",
      "reference": "MS0489",
      "etaUpdate": true,
      "affectedCustomerCount": 37,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "a8f20759-fa95-405b-82a7-856944dccb43",
      "minLongitude": -1.6349,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.60682266118206,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-10-21T15:45:00.000+0000",
      "estimatedRestoration": "2023-10-22T07:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MS0489'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MS0489",
      "maxLongitude": -1.5797,
      "minLatitude": 51.7223,
      "name": "OX18 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "OX18 1PD",
        "OX18 1PE",
        "OX18 2PN",
        "OX18 2PP",
        "OX18 2PU",
        "OX18 2PZ",
        "OX18 2QJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.6349,
              51.7622
            ],
            [
              -1.5797,
              51.7622
            ],
            [
              -1.5797,
              51.7223
            ],
            [
              -1.6349,
              51.7223
            ],
            [
              -1.6349,
              51.7622
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-10-21T16:05:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.7622,
      "etrUpdate": true
    },
    {
      "latitude": 50.83194343840735,
      "type": "LV",
      "reference": "MS0649",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-10-21T19:30:00.000+0000",
      "affectedCustomerCount": 26,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "5069f271-ae1b-4514-8117-0dc18cb163f5",
      "minLongitude": -1.3966,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3899057230594964,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-10-21T17:54:00.000+0000",
      "estimatedRestoration": "2023-10-22T04:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MS0649'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MS0649",
      "maxLongitude": -1.3853,
      "minLatitude": 50.8298,
      "name": "SO45 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO45 2NG",
        "SO45 2NS",
        "SO45 2QE",
        "SO45 2QF",
        "SO45 2QJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3966,
              50.836
            ],
            [
              -1.3853,
              50.836
            ],
            [
              -1.3853,
              50.8298
            ],
            [
              -1.3966,
              50.8298
            ],
            [
              -1.3966,
              50.836
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-10-21T19:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.836,
      "etrUpdate": true
    },
    {
      "latitude": 50.98592181659495,
      "type": "HV",
      "reference": "MS0703",
      "etaUpdate": true,
      "affectedCustomerCount": 12,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "238df49a-91cb-4ccd-a381-d2e42febc1c9",
      "minLongitude": -0.5504,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.5417203910452808,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-10-21T18:58:00.000+0000",
      "estimatedRestoration": "2023-10-22T03:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MS0703'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MS0703",
      "maxLongitude": -0.53,
      "minLatitude": 50.9769,
      "name": "RH20 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RH20 1JP"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.5504,
              50.9899
            ],
            [
              -0.53,
              50.9899
            ],
            [
              -0.53,
              50.9769
            ],
            [
              -0.5504,
              50.9769
            ],
            [
              -0.5504,
              50.9899
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.9899,
      "etrUpdate": true
    },
    {
      "latitude": 56.72441350914568,
      "type": "HV",
      "reference": "MR7371",
      "etaUpdate": true,
      "affectedCustomerCount": 241,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "c5794752-b221-4fe4-ae0e-fca12a0d9a33",
      "minLongitude": -2.7511,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.653830837529654,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-10-19T14:46:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MR7371'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MR7371",
      "maxLongitude": -2.6406,
      "minLatitude": 56.705,
      "name": "DD9 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DD9 6SN",
        "DD9 7DL",
        "DD9 7EL",
        "DD9 7EP",
        "DD9 7ER",
        "DD9 7ES",
        "DD9 7ET",
        "DD9 7EU",
        "DD9 7EW",
        "DD9 7EY",
        "DD9 7EZ",
        "DD9 7HD",
        "DD9 7LA",
        "DD9 7LZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.7511,
              56.7283
            ],
            [
              -2.6406,
              56.7283
            ],
            [
              -2.6406,
              56.705
            ],
            [
              -2.7511,
              56.705
            ],
            [
              -2.7511,
              56.7283
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 56.7283,
      "etrUpdate": true
    },
    {
      "latitude": 57.10990672134335,
      "type": "LV",
      "reference": "MS0775",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-10-21T22:05:00.000+0000",
      "affectedCustomerCount": 16,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "8d5b4521-de85-461e-84ce-657baab2a7f9",
      "minLongitude": -2.2167,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.206304711473682,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-10-21T19:54:00.000+0000",
      "estimatedRestoration": "2023-10-22T03:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MS0775'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MS0775",
      "maxLongitude": -2.1935,
      "minLatitude": 57.1006,
      "name": "AB15 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "AB15 9DR",
        "AB15 9DT",
        "AB15 9EA",
        "AB15 9EE",
        "AB15 9EN",
        "AB15 9EQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.2167,
              57.1133
            ],
            [
              -2.1935,
              57.1133
            ],
            [
              -2.1935,
              57.1006
            ],
            [
              -2.2167,
              57.1006
            ],
            [
              -2.2167,
              57.1133
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-10-22T22:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 57.1133,
      "etrUpdate": true
    },
    {
      "latitude": 60.26659592811345,
      "type": "HV",
      "reference": "MS0846",
      "etaUpdate": true,
      "affectedCustomerCount": 10,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "e70662c7-9e5a-457f-8fe2-9886262d2d11",
      "minLongitude": -1.4057,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3844557210803659,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-10-21T22:16:00.000+0000",
      "estimatedRestoration": "2023-10-22T01:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MS0846'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MS0846",
      "maxLongitude": -1.3441,
      "minLatitude": 60.2445,
      "name": "ZE2 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "ZE2 9LS",
        "ZE2 9NW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.4057,
              60.2843
            ],
            [
              -1.3441,
              60.2843
            ],
            [
              -1.3441,
              60.2445
            ],
            [
              -1.4057,
              60.2445
            ],
            [
              -1.4057,
              60.2843
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 60.2843,
      "etrUpdate": true
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-10-20T09:29:00.000+0000",
      "latitude": 56.38592225972324,
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MR7976'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MR7976",
      "reference": "MR7976",
      "maxLongitude": -3.4461,
      "etaUpdate": true,
      "minLatitude": 56.3813,
      "name": "PH2 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "PH2 0DP",
        "PH2 0EN",
        "PH2 0EW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -3.4526,
              56.3879
            ],
            [
              -3.4461,
              56.3879
            ],
            [
              -3.4461,
              56.3813
            ],
            [
              -3.4526,
              56.3813
            ],
            [
              -3.4526,
              56.3879
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "276758c3-be45-4e9d-be5e-9882e6b69a0c",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -3.4526,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 56.3879,
      "etrUpdate": true,
      "longitude": -3.4484549435549843,
      "resolved": false
    },
    {
      "latitude": 56.52521172300244,
      "type": "HV",
      "reference": "MS0902",
      "etaUpdate": true,
      "affectedCustomerCount": 252,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "9d62b31d-fb52-4d41-b335-50ab26429fb5",
      "minLongitude": -6.9821,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -6.794961115419359,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-10-22T00:30:00.000+0000",
      "estimatedRestoration": "2023-10-22T03:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MS0902'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MS0902",
      "maxLongitude": -6.7095,
      "minLatitude": 56.4554,
      "name": "PA77 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PA77 6AB",
        "PA77 6TN",
        "PA77 6TP",
        "PA77 6TR",
        "PA77 6TS",
        "PA77 6TT",
        "PA77 6TW",
        "PA77 6UA",
        "PA77 6UH",
        "PA77 6UU",
        "PA77 6UY",
        "PA77 6XA",
        "PA77 6XQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -6.9821,
              56.5573
            ],
            [
              -6.7095,
              56.5573
            ],
            [
              -6.7095,
              56.4554
            ],
            [
              -6.9821,
              56.4554
            ],
            [
              -6.9821,
              56.5573
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-10-22T01:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 56.5573,
      "etrUpdate": true
    }
  ]
}
